### PR TITLE
Forge version notifier and command

### PR DESCRIPTION
**Pull request by antoinegag**
# New features
## Forge version command
Running `!mmd forgev` will give you the latest and recommended forge versions for the **latest** Minecraft version. (Can be extended later to query specific versions)
## Forge update notifier
Look for change in latest/recommended forge version for the **latest** Minecraft version. (Can be extended later to query all or specific versions)

# Output:
image

# Other
I added JUnit 5 to the project to easily test components.

### CODE REVIEW

File: src/main/java/com/mcmoddev/bot/helpers/forge/ForgeVersionHelper.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                } else {
+                    version.setLatest(forge);
+                }
+                versions.put(meta.version, version);
+            }
+        }
+
+        // For some reason there's an entry named
+        // latest-1.7.10 from the API, remove it.
+        versions.remove("latest");
```
Comment (jriwanek) [inline src/main/java/com/mcmoddev/bot/helpers/forge/ForgeVersionHelper.java]: Is this supposed to say latest, or latest-1.7.10 like the comment says?

File: src/test/java/com/mcmoddev/bot/helpers/forge/ForgeVersionHelperTest.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    }
+
+    @Test
+    void readVersions() {
+        try {
+            Map<String, ForgeVersion> result = ForgeVersionHelper.getForgeVersions();
+
+            assertTrue(result.size() > 0);
+            assertTrue(result.containsKey("1.7.10"));
+            assertEquals(result.get("1.7.10").getLatest(), "10.13.2.1343");
```
Comment (jriwanek) [inline src/test/java/com/mcmoddev/bot/helpers/forge/ForgeVersionHelperTest.java]: According to the forge site, latest is 1614, error, or API wierdness?

Comment (antoinegag) [reply src/test/java/com/mcmoddev/bot/helpers/forge/ForgeVersionHelperTest.java]: Looks like I missed a special case. The API is messed up.

Result:
image

the second `1.7.10-latest-1.7.10` is overriding it :man_facepalming:

File: src/main/java/com/mcmoddev/bot/helpers/forge/ForgeVersionHelper.java (hunk 1 above)
Comment (antoinegag) [reply src/main/java/com/mcmoddev/bot/helpers/forge/ForgeVersionHelper.java]: The version is parsed with a regex that matches `(.*)-(.*)` so it matches this random `latest-1.7.10` and parse "latest" as the version and puts it in the `version` map with the key "latest", that's why it's simply "latest".

image

File: src/test/java/com/mcmoddev/bot/helpers/forge/ForgeVersionHelperTest.java (hunk 2 above)
Comment (antoinegag) [reply src/test/java/com/mcmoddev/bot/helpers/forge/ForgeVersionHelperTest.java]: Fixed in 8d287355bddfeffe0d633789fb3446b3f665c2ce, good catch!

**Review by KiriCattus — COMMENTED** [review]
Everything works fine after a quick test, any more pointers from you @jriwanek or do you think we can merge now?

**Review by jriwanek — APPROVED** [review]

**Pull request merged**